### PR TITLE
Update foreman_ansible dependency to ~> 6.0

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: foreman_leapp.gemspec [hunk 1]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.test_files = Dir['test/**/*']
 
   s.add_dependency 'foreman_remote_execution', '>= 3.2', '< 5.0'
-  s.add_dependency 'foreman_ansible', '~> 5.0'
+  s.add_dependency 'foreman_ansible', '~> 6.0'
```
Comment (adamruzicka) [inline foreman_leapp.gemspec:23]: Shouldn't we switch to requiring a specific lower version and leaving the upper version unbounded? That's what we do in rex when depending on tasks and in ansible when depending on rex.

Comment (stejskalleos) [reply foreman_leapp.gemspec:23]: You mean like:
`'foreman_ansible', '>= 6.0'`

Comment (adamruzicka) [reply foreman_leapp.gemspec:23]: More like `'foreman_ansible', '>= 5.0'`, since the breaking change there was just a required foreman version bump

**Review by adamruzicka — APPROVED** [review]
ACK

**Comment by adamruzicka** [comment]
Thank you @stejskalleos !

**Pull request merged**